benches/payout-stakers/src/main.rs

```diff
@@ -236,33 +275,35 @@ async fn nominate_validator(
         .zip(iter::repeat(&nominee_account).cloned())
         .collect::<Vec<_>>();
     for chunks in nominator_nominee_accounts.chunks(NOMINATE_CALL_BATCH_LIMIT) {
-        connection
+        root_connection
             .batch_nominate(chunks, TxStatus::InBlock)
             .await
             .unwrap();
     }
 }
 
+
+
 /// Bonds the funds of the validators.
 /// Chooses controller accounts for the corresponding validators.
-/// We assume stash == validator != controller.
-async fn bond_validators_funds_and_choose_controllers(address: &str, validators: Vec<KeyPair>) {
-    let mut handles = vec![];
-    for validator in validators {
-        let validator_address = address.to_string();
-        handles.push(tokio::spawn(async move {
-            let connection = SignedConnection::new(&validator_address, validator).await;
-            connection
-                .bond(MIN_VALIDATOR_BOND, TxStatus::InBlock)
-                .await
-                .unwrap();
-        }));
+/// We assume stash == validator == controller.
+async fn bond_validators_funds_and_choose_controllers(address_ip_without_port: &str, validators: Vec<KeyPair>) {
```

Comment:
Is `and_choose_controllers` still in any way accurate here?

Reply:
Good point, not really, I'll fix naming